osu.Game/Overlays/FirstRunSetup/ScreenUIScale.cs

```diff
@@ -166,7 +168,8 @@ private void load(AudioManager audio, TextureStore textures, RulesetStore rulese
                             {
                                 RelativePositionAxes = Axes.Both,
                                 Position = new Vector2(0.5f),
-                            })
+                            }),
+                            (typeof(ScreenFooter), footer = new ScreenFooter()),
```

Comment:
Maybe comment on these that they are temporary until a global footer is present?

Reply:
This one is a special case, we want to detach the `SongSelect` nested in this screen away from the global footer, because the global footer is already used to present the footer content for the first-run setup overlay. This is exactly like how the `OsuLogo` is cached here right above this line.

Reply:
right, makes sense in context.